src/tile/tile.scss

```diff
@@ -313,3 +320,16 @@
     @include visually-hidden;
   }
 }
+
+@mixin iui-responsive-grid {
```

Comment:
This would be easy to reuse in Layouts grid:
- full row item could have `grid-column: 1 / -1`
- full column item is more dificult to get in implicit grid but I'm not sure how much users would want that.

Maybe move it to separate file if we are going to use this mixing outide tiles?